### DISCUSSION TITLE
No instructions to install deployKF on website as well as github

### DISCUSSION THREAD

**Discussion by Shubham-khapre** [deployKF/deployKF#179] [Questions & Answers] [answered]
where are the instruction to install it ?

**Comment by thesuperzapper** [comment, marked as answer]
There are currently two primary guides which cover installing deployKF:

1. [Local Quickstart](https://www.deploykf.org/guides/local-quickstart/) (for testing deployKF locally in non-production)
2. [Getting Started](https://www.deploykf.org/guides/getting-started/) (for using deployKF in production)

There are many other guides about configuring/using deployKF, but you should start with one of those two.
